src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -300,125 +301,123 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/tools/index';
-@import '../../scss/tools/typography';
-@import '../../scss/settings/colors/index';
+<style>
+@import '../../styles/tools/ec-dropdown';
+@import '../../styles/tools/typography';
 
-$ec-dropdown-search-background-color: $white !default;
-$ec-dropdown-search-background-color-hover: $level-7-backgrounds !default;
-$ec-dropdown-search-background-color-selected: $level-4-tech-blue !default;
-$ec-dropdown-search-background-color-disabled: $ec-dropdown-search-background-color !default;
-$ec-dropdown-search-color: null !default;
-$ec-dropdown-search-color-hover: inherit !default;
-$ec-dropdown-search-color-selected: $white !default;
-$ec-dropdown-search-color-disabled: $level-6-disabled-lines !default;
-$ec-dropdown-search-border-color: $level-6-disabled-lines !default;
-$ec-dropdown-search-icon-color: currentColor !default;
-$ec-dropdown-search-icon-size: 16px !default;
-$ec-dropdown-search-item-height: 40px !default;
-$ec-dropdown-search-item-delimiter-size: 1px !default;
+:root {
+  --ec-dropdown-search-background-color: theme('colors.gray.8');
+  --ec-dropdown-search-background-color-hover: theme('colors.gray.7');
+  --ec-dropdown-search-background-color-selected: theme('colors.key.4');
+  --ec-dropdown-search-background-color-disabled: theme('colors.gray.8');
+  --ec-dropdown-search-color-hover: inherit;
+  --ec-dropdown-search-color-selected: theme('colors.gray.8');
+  --ec-dropdown-search-color-disabled: theme('colors.gray.6');
+  --ec-dropdown-search-border-color: theme('colors.gray.6');
+  --ec-dropdown-search-icon-color: currentColor;
+  --ec-dropddown-search-icon-margin-right: 8px;
+  --ec-dropdown-search-icon-size: 16px;
+  --ec-dropdown-search-item-height: 40px;
+  --ec-dropdown-search-item-delimiter-size: 1px;
+  --ec-dropdown-search-item-horizontal-padding: 8px;
+  --ec-dropdown-search-item-vertical-padding: 16px;
+}
```

Comment:
We should not be creating these variables anymore and just use TW directly: `@apply tw-text-gray-X`.

Reply:
- [ ] use TW classes

Reply:
I only left the variables that are used in calculations.

Reply:
:+1: Better, can we initialize these values from TW theme then? e.g. `theme('spacing.8')`. All these values are available in TW config.

Reply:
- [ ] set using theme